broker/setup_broker/lvk/setup_broker.sh

```diff
@@ -8,7 +8,9 @@ teardown="${2:-False}"
 # "True" tearsdown/deletes resources, else setup
 survey="${3:-lvk}"
 # name of the survey this broker instance will ingest
-region="${4:-us-central1}"
+observation_run="${4:-4}"
+versiontag="O${observation_run}" # 4 -> O4
```

Comment:
This line should prepend a "v" for "version". Does LVK call this schema version "O4"? If so, that is the value that should be passed in as `schema_version` by the user. This line of code should convert that to "vO4" so that we end up with a table named with the syntax `<survey>.alerts_v<schema_version>`. This probably needs to be changed in other places as well.

```suggestion
versiontag="v${observation_run}" # 4 -> v4
```

Reply:
GCN's [Schema Browser](https://gcn.nasa.gov/docs/schema/v4.1.0/gcn/notices#schema-browser) specifies the schema versions for GCN notices but it does not include an explicit schema version specifically for `lvk`. Since I couldn't find an "official" schema version I thought specifying the observation run would be more descriptive. 

I think the user should be specifying the schema associated with the observation run as you bring up: `schema_version="O4"`. What do you think @troyraen?

Reply:
I don't want to tie the observing run to the schema version if that's not what LVK has done. Yes that variable is and should be the schema version. If LVK really hasn't set one, then maybe we don't need this variable here?

Reply:
Agreed. I've removed `schema_version` and `versiontag`